libs/sdk-model/api/sdk-model.api.md

```diff
@@ -868,7 +868,7 @@ export interface IDashboardAttributeFilterConfig {
 // @beta
 export interface IDashboardAttributeFilterParent {
     filterLocalIdentifier: string;
-    over: {
+    over?: {
```

Comment:
Breaking change.
What about this - instead of making this property optional, keep it required, and pass there just empty attibutes array?

Reply:
Ok let's do it this way. But now the question is whether we store this also in the metadata? It is free form so there should be no problem with that. Other option would be to strip it and glue it back in the convertors (which I don't really like).

Reply:
Please create a followup ticket for the making this optional (we will include it in the next major version)